content/post/GSOC_AMA_report.md

```diff
@@ -0,0 +1,105 @@
+
+1.  **Mentoring Sessions**
+    1.  Mentoring students throughout the year through various channels like WhatsApp Members Groups, Workshops, Open Meetups, Casual Meetups, and more.
+    2.  Special Sessions by:
+        1.  **Shivam Rajput** (2017 GSoC Student and Amity Alumni)
+        2.  **Shashank** (Debian GSoC Student)
+        3.  **Raghav Puri** (Organization Admin and Mentor, AOSSIE Google Code-In, and GSoC 2019)
+        4.  **Anamika** (Systers Admin)
+        5.  **Pavini** (RGSoC Student)
+        6.  **Aunvrat Parashar** (Founder, ALiAS)
+        7.  **Shyam Saini** (GSoC Student 2017)
+        8.  **Vipul Gupta** (GSoC Student 2018, ALiAS founding member),
+
+2.  **AMA Sessions**
+ALiAS Conducted **15 days 15 sessions of Ask Me Anything, on GSoC** in the last two weeks before GSoC Proposal Deadline, to help out students and guide them on various aspects of GSoC.
+    
+3.  **Webinars**
+* [GSoC'19 Live\] Drafting the Perfect Proposal with [realslimshanky](https://twitter.com/realslimshanky) | ALiAS
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/HlvI9lX2U0g" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+*   GSoC Ask Me Anything with Raghav Puri \[Live\] ([**https://www.youtube.com/watch?v=0W9BuKwm\_h4**]()**)**
+
+The joint efforts of various students, mentors, leaders, trainers, volunteers, and contributors have led us to this proud moment.
+
+ALiAS has always been helping students contribute to open source and will continue to do so.
+
+## Social Media Reach
+
+
+![](https://lh5.googleusercontent.com/F5vcpGwKq8O5PzH4jP6DNzf7XUHqZTfYr_13o4twRGKabJIjQsNsoItZi8WrE0IAXsUd6zXFj4FDNJACRD5V0Klvq0VPVT0AWnN0ehlSQ2_yExCrZj3TGqv2k1FU3x_JoTqXwQM)
+
+![](https://lh4.googleusercontent.com/-6Sp9_xkf1QNwZUxi00pEv--hO03qLh0N1Q3ZcqpMFUeclCcpFCcFj29Ha7NqNxOHm1-5Y4WT-cWRF84VedBAF9KbGH3jtJNQq0HVd_RsukXc8QZUHkC1mBaPYWheMgN4xNk6P0)
+
+![](https://lh5.googleusercontent.com/VajPqpPjWA8hKd1iXznPGTXo5xnlRv8g2O6T2wYohXovi_nXUIu2Pcog9wjyYM269kF6ELx-U5AmsbPhbJd7x_szWi9xnoMGLwyRXAbTMncoFITbG5xDoKPrXK_NjFlCXbyBjbo)
+
+## Feedback from the community!
+
+|||| 
+|--- |--- |--- | 
+|Name|Feedback|Selected?| 
+|Parth Sharma|The whole alias community with special thanks to Vipul|Yes| 
+|Nikhil Maan|Thank you Vipul Gupta Bhaiya and Rohan Verma for the reviews and everyone present at the AMAs for the discussions.Also, special thanks to Vipul Bhaiya again for going out of the way to edit some parts of the proposal himself.|Yes| 
+|Ayush Agarwal|Thanks to Vipul in writing the proposal.|No| 
+|Omkar Yadav||No| 
+|Akshat Pande|Thank you so much, everyone, especially Mr. Vipul gupta❤️|No| 
+|Priyansh Chaudhary|Thanks to Vipul and ALiAS|No| 
+|Saurav|Yes, Vipul Gupta helped me in reviewing my proposal, thanks a lot to Vipul Gupta and all the members of ALiAS.|No| 
+|Sahil|yes, the AMA sessions cleared my lot of doubts|No| 
+|Harsh Prasad|Thanks to Animesh Bhaiya|No|
+
```

Comment:
There were many more names in the report. What happened?

Reply:
There were empty rows with just names, kept rows with feedback only.

Reply:
I think we should add the names of people who tried too.

Reply:
Will add more.